iam/terraform/account-specific/main/regional-log-subscription-filters/create-missing-log-groups.sh

```diff
@@ -7,8 +7,10 @@
 
 set -e
 
-ENVIRONMENTS=$@
-if [[ -z "${ENVIRONMENTS}" ]]; then
+ENVIRONMENT_STR=$*
+IFS=' ' read -r -a ENVIRONMENTS <<< "$*"
```

Comment:
Specifically not supporting space delimited environments?  Hence the `$*`?

Also, can you do this?
```suggestion
ENVIRONMENT_STR=$*
IFS=' ' read -r -a ENVIRONMENTS <<< "${ENVIRONMENT_STR}"
```

Reply:
can you re-review this file?  I was trying to fix a shellcheck error but I don't think I was doing the best approach.